src/cli.cr

```diff
@@ -308,17 +309,17 @@ module DppmRestApi::CLI
                 else
                   STDOUT
                 end
-    api_key = Random::Secure.base64 24
+    group_list = split_numbers groups
+    api_key, user = DppmRestApi::Config::User.create group_list, name
     output_io.puts "API key for user named '#{name}'"
     output_io.puts api_key
     if using_stdout
+      print "ok? (Y/n) "
       raise "cancelled" if gets(chomp: true).try &.downcase.starts_with?('n')
     else
```

Comment:
You can use `DPPM::CLI.confirm_prompt { raise "cancelled." }`

Reply:
Nice!